Makefile.uk

```diff
@@ -130,7 +130,6 @@ LIBNEWLIBGLUE_SRCS-y += $(LIBNEWLIBC_BASE)/console.c
 LIBNEWLIBGLUE_SRCS-y += $(LIBNEWLIBC_BASE)/plat.c
 LIBNEWLIBGLUE_SRCS-y += $(LIBNEWLIBC_BASE)/file.c
 LIBNEWLIBGLUE_SRCS-y += $(LIBNEWLIBC_BASE)/mem.c
-LIBNEWLIBGLUE_SRCS-y += $(LIBNEWLIBC_BASE)/resource.c
```

Comment:
I understand that this PR aims to exclude this source from the build process so that Simon's PR [1] can be merged.
Shouldn't we also delete the resources.c source from the newlib repo?
[1] https://github.com/unikraft/unikraft/pull/174